### PR TITLE
add documentation to help contribute

### LINKED ISSUE #447: Document how to do project setup when someone starts to contribute
I've done a fork of the project and I don't know how can I install the things to start contributing.

### PR DESCRIPTION
**Pull request by VeroGarp**
## Description
Documentation to help contribute

## Changes
Only documentation

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [x] Updated documentation accordingly
 
## Additional information
Closes #447

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
 
 Make sure that you assign the chosen issue to yourself to communicate your intention to work on it and reduce the possibilities of other people taking the same assignment.
 
+### Getting the code
+
+To start contributing to the project you must first follow these steps in your terminal.
```
Comment (NickSeagull) [inline CONTRIBUTING.md:123]:
```suggestion
To start contributing to the project you would need to set up the project in your system, to do so, you must first follow these steps in your terminal.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Install Yarn with npm `install -g yarn`
```
Comment (NickSeagull) [inline CONTRIBUTING.md:125]:
```suggestion
- Install Yarn: `npm install -g yarn`
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Install Lerna with npm `install -g lerna`
```
Comment (NickSeagull) [inline CONTRIBUTING.md:127]:
```suggestion
- Install Lerna: `npm install -g lerna`
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Clone the repo and cd to the project
```
Comment (NickSeagull) [inline CONTRIBUTING.md:129]:
```suggestion
- Clone the repo and get into the directory of the project: `git clone <WRITE REPO URL HERE> && cd booster`
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Write `lerna bootstrap`
```
Comment (NickSeagull) [inline CONTRIBUTING.md:131]:
```suggestion
- Install project dependencies: `lerna bootstrap`
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Write `lerna run compile`
```
Comment (NickSeagull) [inline CONTRIBUTING.md:133]:
```suggestion
- Compile the project `lerna run compile`
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Now you can write your contribution and to make sure everything works you whould write lerna run test
```
Comment (NickSeagull) [inline CONTRIBUTING.md:135]:
```suggestion
- Add your contribution

- Make sure everything works by executing the unit tests: `lerna run test`
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -117,6 +118,24 @@ Both issue lists are sorted by the total number of comments. While not perfect,
...
+
+- Before make a PR you should execute `./scripts/check-all-the-things.sh` on Linux and MacOs, but if you are in Windows you should execute `.\scripts\check-all-the-things.ps1`
```
Comment (NickSeagull) [inline CONTRIBUTING.md:137]:
```suggestion
- Before making a PR you should run the `check-all-the-things` script:
  - `./scripts/check-all-the-things.sh` on Linux and MacOS
  - `.\scripts\check-all-the-things.ps1` on Windows
```

**Review by NickSeagull — CHANGES_REQUESTED** [review]
Hello, thanks for contributing! Left some suggestions

**Comment by VeroGarp** [comment]
Thanks for the suggestions, I still have a lot to improve! @NickSeagull 😊

File: CONTRIBUTING.md (hunk 3 above)
Comment (NickSeagull) [reply CONTRIBUTING.md:127]: @VeroGarp remember to accept this suggeston

**Review by NickSeagull — APPROVED** [review]
Thanks!!!

**Review by javiertoledo — APPROVED** [review]
Very useful!

**Pull request merged**